region/lowernorfair/west.json

```diff
@@ -1033,13 +1330,24 @@
           "to": [
             {
               "id": 2,
-              "requires": [
-                {"or": [
-                  "heatProof",
-                  "canHeatRun"
-                ]},
-                "canOpenGreenDoors",
-                {"heatFrames": 600}
+              "strats": [
+                {
+                  "name": "Base",
+                  "notable": false,
+                  "requires": [
+                    "canNavigateHeatRooms",
+                    "canOpenGreenDoors",
+                    {"heatFrames": 600},
+                    {"or":[
+                      "ScrewAttack",
+                      {"enemyDamage": {
```

Comment:
power bomb or super kills should save health over taking contact hits.  
I'm not sure how helpful ice or dboosts are.
CanComeInCharged for sure, if we're doing that.

Reply:
I think my rationale was that trying to kill them was liable to cost as much in heat frames as the energy of charging through.

I'm reconsidering now because heatproof scenarios are different.

Reply:
yeah, with heatproof it makes sense.  I didn't realize their contact damage was so low that it doesn't matter much without heatproof.  Super kills save ~10 health, PB kills lose more health but gain drops.  I did find a slick dboost (left to right) however.